extensions/kafka/src/main/java/io/deephaven/kafka/ingest/KafkaStreamConsumer.java

```diff
@@ -13,5 +13,12 @@
  * Consumer for lists of ConsumerRecords coming from Kafka. The StreamFailureConsumer is extended so that we can report
  * errors emanating from our consumer thread.
  */
-public interface KafkaStreamConsumer extends Consumer<List<? extends ConsumerRecord<?, ?>>>, StreamFailureConsumer {
+public interface KafkaStreamConsumer extends StreamFailureConsumer {
+    /**
+     * Consume a list of ConsumerRecords coming from Kafka.
+     *
+     * @param records the records to consume
+     * @return the total number of message bytes processed
```

Comment:
Not sure if worth clarifying a bit more - but it's the sum of *serialized* key and value sizes of all the records. I _think_ this means that the in-memory representation may be much bigger (depending on if there is wire compression). But I also _think_ it means that there are bytes unaccounted for (headers, framing, etc).